CHANGES

```diff
@@ -9,6 +9,11 @@
 
  Changes between 1.1.0f and 1.1.1 [xx XXX xxxx]
 
+  *) Add "atfork" functions.  If building on a system that without
+     pthreads, see doc/man3/OPENSSL_fork_prepare.pod for application
+     requirements.  The RAND facility now uses/requires this.
```

Comment:
There is ambiguity in this statement. It suggests that *all* systems have fork() and some of them don't have pthreads. There are systems/environments that have pthreads, but don't actually have fork. But more importantly try to find fork-capable yet pthreads-incapable system...